Dockerfile.prod

```diff
@@ -20,8 +25,8 @@ EOT
 ENV DEBUG_MODE off
 # Set chameleon cache directory
 ENV CHAMELEON_CACHE /app/var/cache
-# Enable compilation of po files into mo files
-ENV zope_i18n_compile_mo_files true
+# Disable compilation of po files into mo files (This should be done in the builder image)
+ENV zope_i18n_compile_mo_files=
```

Comment:
But if you add custom packages/addons to your production or development images, you'll need to compile mo files for those as well after/based on the builder image.

Reply:
Yep, how would we users of derived images do this?